src/main/java/com/minecolonies/coremod/commands/colonycommands/ColonyTeleportCommand.java

```diff
@@ -1,27 +1,36 @@
 package com.minecolonies.coremod.commands.colonycommands;
 
+import static com.minecolonies.coremod.commands.AbstractSingleCommand.Commands.COLONYTP;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import javax.annotation.Nullable;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.annotation.Nullable'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)